addons/website_event_booth_sale/models/sale_order.py

```diff
@@ -53,6 +53,6 @@ def _cart_update(self, product_id=None, line_id=None, add_qty=0, set_qty=0, **kw
         product = self.env['product.product'].browse(product_id)
         if product.type == 'event_booth' and set_qty > 1:
             set_qty = 1
-            values['warning'] = _('Sorry, you can\'t modify quantity to an Event Booth product.')
+            values['warning'] = _('You cannot manually change the quantity of an Event Booth product.')
```

Comment:
In your commit message ("polish UX design") you mention that you:
> - fix the bug that allowed the user to modify quantities

Is that this line? I'm not sure I understand exactly.
It seems to only modify the warning message.

Reply:
No indeed... I split the commit so the fix isn't there anymore ^^
I removed this point from the commit message !